data_diff/dbt.py

```diff
@@ -324,12 +324,7 @@ def _cloud_diff(diff_vars: DiffVars, datasource_id: int, datafold_host: str, url
             send_event_json(event_json)
 
         if error:
-            rich.print(
-                "[red]"
-                + ".".join(diff_vars.prod_path)
-                + " <> "
-                + ".".join(diff_vars.dev_path) + "[/]\n"
-            )
+            rich.print("[red]" + ".".join(diff_vars.prod_path) + " <> " + ".".join(diff_vars.dev_path) + "[/]\n")
```

Comment:
Should we use an f-string to keep it consistent with the print statements below?

Reply:
yes I'll fix that in this PR https://github.com/datafold/data-diff/pull/464/files

Reply:
done